docs/_docs/help/troubleshooting.md

```diff
@@ -22,17 +22,15 @@ check out the following tips:
 
 *Node Version*
 
-Verify that you are using the correct node version by running:
+Use the command-line to verify that you are using Node version 5.10.0 or higher by running:
```

Comment:
Does it make sense to be less specific on the version here and say like we do in another places...

> A version of Node that is equal to or greater than that specified in [“node” (under “engines”) in the package information](https://github.com/facebook/nuclide/blob/master/package.json)